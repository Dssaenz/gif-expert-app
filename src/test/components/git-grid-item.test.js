import React from 'react'
// Dependencies
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

//Components
import GifGridItem from '../../components/git-grid-item';

describe('Test in <GifGridItem /> component' , () => {

	const title = 'Default title';
	const url = 'https://localhost/algo.jpg';

	test('It should display the component correctly', () => {
		const wrapper = shallow(<GifGridItem title={title} url={url} />)
		expect( wrapper ).toMatchSnapshot();
	})
})