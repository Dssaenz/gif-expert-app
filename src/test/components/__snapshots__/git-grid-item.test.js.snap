// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <GifGridItem /> component It should display the component correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <p>
    Default title
  </p>
  <img
    alt="Default title"
    src="https://localhost/algo.jpg"
  />
</div>
`;
